### PR TITLE
Improve stability of Application Operator Tests

### PR DESCRIPTION
**Pull request by Szymongib**
**Description**

Changes proposed in this pull request:

- Event Service tests - Wait for Istio Sidecar to be up before starting tests
- Application Gateway Tests - Use resilient client

**Related issue(s)**

### CODE REVIEW

File: tests/integration/event-service/test/apitests/gateway_events_test.go [hunk 1]
```diff
@@ -42,9 +42,6 @@ const (
 
 func TestGatewayEvents(t *testing.T) {
 
-	config, err := testkit.ReadConfig()
-	require.NoError(t, err)
```
Comment (Abd4llA) [inline tests/integration/event-service/test/apitests/gateway_events_test.go]: Can you give more context why would it be ok for ReadConfig to return an error ?

Comment (Szymongib) [reply tests/integration/event-service/test/apitests/gateway_events_test.go]: It is not. Reading config was moved to the `TestMain` function as it needs to be run only once https://github.com/kyma-project/kyma/pull/5890/files#diff-5d812308d56ccc84812557a61b547728R25

**Pull request merged**